paymentsheet/src/main/java/com/stripe/android/paymentsheet/forms/TransformSpecToElement.kt

```diff
@@ -75,6 +54,32 @@ private fun transform(
     )
 }
 
+private fun transform(
+    sectionFields: List<SectionFieldSpec>,
+    focusRequesterCount: FocusRequesterCount
+) = sectionFields.map {
```

Comment:
In this case it might be nice to explicitly include the return type.

Reply:
will do